src/lazy/expanded/macro_evaluator.rs

```diff
@@ -1206,30 +1229,20 @@ impl<'top> TemplateExpansion<'top> {
 mod tests {
     use crate::{v1_1, ElementReader, Int, IonResult, MacroTable, Reader};
 
-    /// Reads `input` and `expected` using an expanding reader and asserts that their output
-    /// is the same.
-    fn eval_enc_expr<'data>(input: &'data str, expected: &'data str) -> IonResult<()> {
-        let mut actual_reader = Reader::new(v1_1::Text, input.as_bytes())?;
+    /// Reads `input` and `expected` and asserts that their output is Ion-equivalent.
+    fn stream_eq<'data>(input: &'data str, expected: &'data str) -> IonResult<()> {
```

Comment:
🗺️ Now that the `Reader` understands encoding directives and we have an `:append_macros` system macro, we no longer need to manually register individual macros by source.